app/views/flow/start.html.erb

```diff
@@ -36,6 +36,6 @@
 
   <p>If you don’t have an account, you’ll need to <%= link_to 'register first', href="#{ENV['OAUTH_SERVER']}/register" %>.</p>
   
-  <p>If you require admin access please <%= link_to 'click here', admin_employees_path %>.</p>
+  <p>If you require admin access please <%= link_to 'click here', eligible_path %>.</p>
```

Comment:
The paths here are a little confusing, the admin login takes me to /eligible.  As a user I would expect /login or /admin. 

I'm thinking we can create a simple view with the same content as eligible.html.erb on the  `/admin` route for admin users.